content/quickstart/kubernetes/tutorial-clusters.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Tutorials - Clusters
+aliases: ["/reference/kubernetes-clusters.html"]
+menu:
+  quickstart:
+    identifier: kubernetes-clusters
+    weight: 6
+    parent: kubernetes
+---
+
+<img src="/images/quickstart/k8s-purple.png" align="right">
```

Comment:
I don't think the k8s logo is allowed to be purple

Reply:
This is an existing image we use elsewhere. We can fix this as part of #1107.